File: pgm/ABC_Dart/ABC_432_A.dart
import 'dart:io';

void main() {
  final List<int> ABC = stdin
      .readLineSync()!
      .split(" ")
      .map(int.parse)
      .toList();
  const String blank = "";
  ABC.sort((a, b) => b.compareTo(a));
  print(ABC.join(blank));
  exit(0);
}

File: pgm/ABC_C#/ABC_432_A.cs
using System;
// using System.Collections.Generic;
// using System.Globalization;
// using System.IO;
// using System.Linq;
// using System.Text;
// using System.Text.RegularExpressions;

class Program
{
    static void Main()
    {
        int[] ABC = Array.ConvertAll(Console.ReadLine().Split(' '), int.Parse);
        Array.Sort(ABC);
        Array.Reverse(ABC);
        Console.WriteLine(string.Concat(ABC));
        Environment.Exit(0);
    }
}